src/api/apis/TeddyCloudApi.ts

```diff
@@ -334,7 +340,7 @@ export class TeddyCloudApi extends runtime.BaseAPI {
         }
 
         const response = await this.request({
```

Comment:
@SciLor : it seems that he fails here and this is not executed:

if (this.isJsonMime(response.headers.get('content-type'))) {
            return new runtime.JSONApiResponse<string>(response);
        } else {
            return new runtime.TextApiResponse(response) as any;
        }